### PR TITLE
Ansible: gluster-brick-create switch from community modules to commands

### PR DESCRIPTION
**Pull request by mnecas**
This is a minor change to the gluster-brick-create ansible role. The main change is that we switched from community modules which are no longer supported to the commands. We need this change for possible ansible-core support.
Along with the changes, I have bit improved the readme and ansible lint for better readability.
I was able to test these changes on my server with multiple reruns and looked fine, but I was not able to test it with gluster.

Bug-Url: https://bugzilla.redhat.com/2052690

### CODE REVIEW

**Comment by mnecas** [comment]
Hi @pkesavap @sabose could you do please review from gluster perspective? 
I was not able to test the changes with gluster, could someone please test the changes?

File: packaging/ansible-runner-service-project/project/roles/gluster-brick-create/README.md [hunk 1]
```diff
@@ -7,32 +7,45 @@ Given a set of physical devices, this role creates a volume group, a thin pool a
...
+| stripesize        | no       | 256     |          | Stripe size configured at RAID controller. Value should be in KB. Required only in case of RAID disk type.
+| vgname            | yes      |         |          | Name of the volume group that the disk is added to. The Volume Group will be created if not already present
+| size              | yes      |         |          | Size of thinpool to be created on the volume group. Size should contain the units. For example, 100G
+| lvname            | yes      |         |          | Name of the Logical volume created using the physical disk(s).
+| ssd               | yes      |         |          | Name of the ssd device.
+| cache_lvname      | yes      |         |          | Name of the Logical Volume to be used for cache.
+| cache_lvsize      | yes      |         |          | Size of the cache logical volume
+| mntpath           | yes      |         |          | Path to mount the filesystem.
+| pool_metadatasize | yes      |         |          | Size of the pools metadata, should be between 2M to 16G (Example: 24M).
```
Comment (barpavel) [inline packaging/ansible-runner-service-project/project/roles/gluster-brick-create/README.md:23]: Just make sure it' shouldn't be `2MiB` & `16GiB`.

**Review by barpavel — APPROVED** [review]
Looks OK :)

**Comment by rchikatw** [comment]
Verified the patch on my setup working as expected. 
LGTM +1

**Review by mwperina — APPROVED** [review]
+1

**Pull request merged**